w3-filecoin.md

```diff
@@ -0,0 +1,187 @@
+# Filecoin Protocol
```

Comment:
`Filecoin Protocol` seems overloaded with the actual protocols that make up Filecoin.

one of Aggregate, Piece, Deal would be better (but not great)

Reply:
We have moved out of Aggregate because of historical namings confusion as well. I see this repo as protocols within web3.storage stack, and therefore Filecoin protocol looks fine. We can rename to `Filecoin deal Protocol` (or similar) if helpful